crates/bevy_pbr/src/material.rs

```diff
@@ -817,6 +818,7 @@ pub fn extract_materials<M: Material>(
                 changed_assets.remove(id);
                 removed.push(*id);
             }
+            AssetEvent::NoLongerUsed { .. } => {}
```

Comment:
```suggestion
            AssetEvent::NoLongerUsed { .. } |
```

(and remove the lint override)

Reply:
I did it that way because we're eventually going to want to handle LoadedWithDependencies differently, but only for now will it have the same match arm..